backend/typescript/middlewares/validators/camperValidators.ts

```diff
@@ -9,123 +9,144 @@ import {
 
 /* eslint-disable @typescript-eslint/explicit-module-boundary-types */
 /* eslint-disable-next-line import/prefer-default-export */
-export const createCamperDtoValidator = async (
+export const createCampersDtoValidator = async (
   req: Request,
   res: Response,
   next: NextFunction,
 ) => {
-  if (!validatePrimitive(req.body.campSession, "string")) {
-    return res.status(400).send(getApiValidationError("campSession", "string"));
+  let campSession = "";
+  let chargeId = "";
+  if (req.body.length > 0) {
+    campSession = req.body[0].campSession;
+    chargeId = req.body[0].chargeId;
   }
```

Comment:
we can change this to return an error if the body length is less than 1 and move the assignment out of the if statement? what do you think 
```suggestion
  if (req.body.length === 0) {
    return res
      .status(400)
      .send("No campers sent - there must be at least one camper in the request.");
  }
  campSession = req.body[0].campSession;
  chargeId = req.body[0].chargeId;

```

Reply:
yea i agree!